ElenaCzubiak-elena-cz/HW_2/Problem5.rb

```diff
@@ -0,0 +1,8 @@
+puts "Check if all elements in an array are Fixnum"
+
+def array_of_fixnums?(array)
+  truth_array = []
+  array.each { |i| truth_array << i.is_a?(Fixnum)}
+  !truth_array.include?(false)
```

Comment:
truth_array.exclude?(false)

I think I had tried "excludes?" instead, thanks for the feedback!